hermes/probe/monitor.go

```diff
@@ -12,9 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+// Authors:
+// - Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
+// - Evan Spendlove, GitHub: evanSpendlove.
+//
 // Monitor implements a probe that monitors a storage system using the
 // Hermes algorithm.
 
+package probe
+
+import (
+	pb "github.com/googleinterns/step224-2020/config/proto"
+)
+
 // MonitorProbe holds aggregate information about all probe runs, per-target.
 // It also holds the config and options used to initialise the probe.
-type MonitorProbe struct{}
+type MonitorProbe struct {
+	name    string
+	config  *pb.HermesProbeDef
+	targets []string
+	opts    *options.Options
```

Comment:
not clear where does `options` come from in here. I think you should include its path in your imports.

Reply:
Could you add a comment of what targets []string are? since there's also config.targets this could be confusing for the user (at least it was for me as you know)

Reply:
I've amended my imports in my latest commit.

Also, I've changes targets to an `[]*pb.Target` as I feel that this will be more useful.